index.html

```diff
@@ -14,23 +14,22 @@
         <img id="logo" src="./images/image_geometry_2.png"  alt="logo">
         <img id="icon" src="./images/hamburguer.png" alt="menu">
     </header>
-    <!--<nav id="navbar">
+    <nav id="navbar">
         <ul id="list">
             <li><a id="home" href="index.html">Home</a></li>
             <li><a id="portfolio" href="portfolio.html">Portfolio</a></li>
             <li><a id="about" href="about.html">About</a></li>
             <li><a id="contact" href="contact.html">Contact</a></li>
             <li><img id="line" src="./images/Rectangle_52.png" alt="rectangle to underline options"></li>
         </ul>
-    </nav>-->
-
+    </nav>
     <main>
         <section id="presentation">
             <article>
                 <p id="name">Francisco Borja</p>
```

Comment:
- [ ] As mentioned before, this heading `Francis Borja` is the main heading of your page so for best semantic practices, it would be good to use an `h1` tag here rather than a `p`. Kindly address this issue.